test/error-handler-system-errors.test.js

```diff
@@ -0,0 +1,144 @@
+'use strict'
+
+const t = require('tap')
+const Assert = require('assert')
+const test = t.test
+const sget = require('simple-get').concat
+const fastify = require('..')()
+
+const systemErrors = {
+  EvalError,
+  RangeError,
+  ReferenceError,
+  SyntaxError,
+  TypeError,
+  URIError,
+  AssertionError: Assert.AssertionError
+}
+
+const requestTimeout = 200
```

Comment:
timeouts in tests are a source of flakiness, can you refactor this?

Reply:
This timeout is for `simple-get` request

Reply:
Fixed